app/experimenter/nimbus-ui/src/components/PageEditAudience/index.test.tsx

```diff
@@ -97,6 +97,22 @@ describe("PageEditAudience", () => {
     });
   });
 
+  it("redirects to the review page if the experiment status is review", async () => {
```

Comment:
nit: I'm a bit confused by "... if the experiment status is review".

1. Should that be something like "in the 'review' state"?
2. Probably just a copy pasta, but should that say "is preview"?
    L70 above already has a test `it("redirects to the review page if the experiment status is review", async () => {` which could be confusing since they share a test name, but set different statuses.

I guess same comment applies to the 3 similar copy-pasta-pasta-pasta's below too.

Reply:
heh, yes good catch. will update.